extensions/wikia/GlobalWatchlist/GlobalWatchlistBot.class.php

```diff
@@ -9,7 +9,8 @@ class GlobalWatchlistBot {
 	const REPLY_ADDRESS = 'noreply@wikia.com';
 
 	public function __construct() {
-		\F::app()->wg->ExtensionMessagesFiles['GlobalWatchlist'] = dirname( __FILE__ ) . '/GlobalWatchlist.i18n.php';
+		$messageFiles = \F::app()->wg->ExtensionMessagesFiles;
+		$messageFiles['GlobalWatchlist'] = dirname( __FILE__ ) . '/GlobalWatchlist.i18n.php';
```

Comment:
The line as is was throwing this error in dev:
```php
Indirect modification of overloaded property WikiaGlobalRegistry::$ExtensionMessagesFiles has no effect in /usr/wikia/source/app/extensions/wikia/GlobalWatchlist/GlobalWatchlistBot.class.php on line 12
```
Even though its just a notice, it was somehow randomly killing requests